src/jdk.jfr/share/classes/jdk/jfr/AnnotationElement.java

```diff
@@ -375,7 +375,7 @@ private static void checkType(Class<?> type) {
         throw new IllegalArgumentException("Only primitives types or java.lang.String are allowed");
     }
 
-    // Whitelist of annotation classes that are allowed, even though
+    // Allow-list of annotation classes that are allowed, even though
```

Comment:
I guess we can just say "list" here:
```suggestion
    // List of annotation classes that are allowed, even though
```